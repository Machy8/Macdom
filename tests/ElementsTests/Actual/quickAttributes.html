a $www.google.com $blank Link on Google
a 3$button Button
a ${plink :Core:Front:Homepage:default}; Link on homepage in Nette Framework
input 2$+=´´)ú¨§ů-.,-*/+_:\\()[]@&|\#'
input $text $Click me!;
input disabled 2$Disabled button; 1$submit
