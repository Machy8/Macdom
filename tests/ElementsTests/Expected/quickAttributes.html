<a href="www.google.com" target="blank">
	Link on Google
</a>
<a role="button">
	Button
</a>
<a href="{plink :Core:Front:Homepage:default}">
	Link on homepage in Nette Framework
</a>
<input value="+=´´)ú¨§ů-.,-*/+_:\\()[]@&|\#'">
<input type="text" value="Click me!">
<input value="Disabled button" type="submit" disabled>
<div id="myId" class="myClass"></div>
<i data-my-data="Some value" data-some-data="Another value"></i>
